admin/listPurchases/listView.html

```diff
@@ -107,7 +107,7 @@
             <td>${entry.customer || 'En kunde'}</td>
             <td class="status">${translateStatus(entry.status)}<br><span class="status-date">${entry.statusChangeDate || "?"}</span></td>
             <td>${badgeImages.map((imageUrl) => `<img class="image-thumbnail" src="${imageBaseUrl + imageUrl}">`).join('')}</td>
-            <td>${entry.isPreorder ? 'PRE: ' : ''}${entry.description.replaceAll(" + ", "<br>").replaceAll(",",":").replace(/\d* stk./g, "<b>$&</b>")}</td>
+            <td>${entry.isPreorder ? 'PRE: ' : ''}${entry.items.map((item) => item.id && item.count ? item.id + ": <b>" + item.count +" stk</b>" : item.description).join("<br>")}</td>
```

Comment:
Extract a function for rendering the entry here, and take it in steps. A suggestion for how to break it down for readability (notice that I try to make the function more linear and limit how many things the reader has to keep in mind while reading):

```js
function describeEntry(entry) {
  const preOrderTag = entry.isPreorder ? 'PRE: ' : '';

  const itemDescriptions = entry.items
    .map((item) => summarizeItem(item))
    .join("<br>");

  return `${preOrderTag}${itemDescriptions}`;
}

function summarizeItem(item) {
    // Detect newer items with formal id and count
    if(item.id && item.count) {
        return `${item.id}: <b>${item.count}</b>`;
    }
    return item.description;
}
```

Refactoring the code into this revealed to me that we are displaying `item.id`, which isn't the human-readable name for the badge if I recall correctly!

So we should load in the display name on each item on the backend, so we can display it here in the frontend.